### PR TITLE
add package: 2bwm (#600)

### PR DESCRIPTION
**Pull request by Yisus7u7**
Close (#600)

### CODE REVIEW

**Comment by Yisus7u7** [comment]
Done

File: packages/2bwm/build.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+TERMUX_PKG_VERSION=0.3
+TERMUX_PKG_SRCURL="https://github.com/venam/2bwm/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256="a4889ea4b01b1a3d4a508daa034b9d86676913cbbca1f977858df692a6e2af95"
+TERMUX_PKG_DEPENDS="libx11, libxcb, libxkbcommon, xcb-util, xcb-util-cursor, xcb-util-keysyms, xcb-util-wm, xcb-util-xrm, xcb-util-renderutil, xcb-util-image, xorgproto"
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_make(){
+	cd ${TERMUX_PKG_SRCDIR}
+	make
+}
```
Comment (xtkoba) [inline packages/2bwm/build.sh:14]: Is `termux_step_make()` necessary to be defined here? I suppose the default one is sufficient. If a parallel build does not work then you can set `TERMUX_MAKE_PROCESSES=1`.

File: packages/2bwm/build.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+termux_step_make_install () {
```
Comment (xtkoba) [inline packages/2bwm/build.sh:16]: Parenthesis of functions should not be preceded with a space, as per <a href="https://github.com/termux/termux-packages/wiki/Coding-guideline">Coding guideline</a>.

File: packages/2bwm/build.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/venam/2bwm"
+TERMUX_PKG_DESCRIPTION="A fast floating WM written over the XCB library and derived from mcwm."
+TERMUX_PKG_LICENSE="Unlicense"
```
Comment (xtkoba) [inline packages/2bwm/build.sh:3]: The license of `2bwm` is not `Unlicense` but `ISC`. See the head of `2bwm.c` for example. The source tarball does not have a separate license file, so you need to extract the license notice from the source and put it in a separate file.

**Comment by Yisus7u7** [comment]
@xtkoba I am somewhat busy with my studies, when I have the free time I do it

**Comment by xtkoba** [comment]
No hurry at all. Mind your job! :+1:

File: packages/2bwm/build.sh (hunk 3 above)
Comment (ar664) [inline packages/2bwm/build.sh:3]:
```suggestion
TERMUX_PKG_LICENSE="ISC"
```

**Pull request closed without merging**